src/components/Menu.js

```diff
@@ -40,20 +40,19 @@ const Menu = () => {
             <Nav.Link
               id={itemId}
               key={i + menuItem.url}
+              href={path}
               className={
                 "menu-item menu-item-type-custom menu-item-object-custom menu-item-home " +
                 itemId
               }
             >
-              <UniversalLink
-                to={path}
-                activeClassName={"current-menu-item current_page_item nav-link"}
```

Comment:
We need to incorporate the current-menu-item/current_page_item so we can tell what page has been selected.

Reply:
Universal Link has been incorporated.